project/AppDependencies.scala

```diff
@@ -2,7 +2,7 @@ import sbt.*
 
 object AppDependencies {
   private lazy val mongoHmrcVersion = "2.1.0"
```

Comment:
running the dependency check locally, I can see the following:

[info] Found 5 dependency updates for binding-tariff-classification
[info]   org.apache.pekko:pekko-actor-typed             : 1.0.2 -> 1.0.3         
[info]   org.apache.pekko:pekko-serialization-jackson   : 1.0.2 -> 1.0.3         
[info]   org.apache.pekko:pekko-stream                  : 1.0.2 -> 1.0.3         
[info]   uk.gov.hmrc.mongo:hmrc-mongo-play-30           : 2.1.0          -> 2.2.0
[info]   uk.gov.hmrc.mongo:hmrc-mongo-test-play-30:test : 2.1.0          -> 2.2.0


could you update those or will there be any dep issue?

Reply:
I did try to update some but then I remember some deps were unable to be pulled in to the project or were not found think there were some small clashes. Will try again and see.